docs/CONTRIBUTING.md

```diff
@@ -555,11 +555,11 @@ For more details about the CI setup, you can check the
 
 #### Running IDE
 
-You can start [IDE](https://github.com/enso-org/enso/tree/develop/gui) with a
-development version of the language server. IDE executable has `--no-engine`
-flag that switches off the bundled backend. That requires you to run the project
-manager process yourself. Running development version of the IDE is also
-possible via the npm script in the root of the repository:
+You can start IDE with a development version of the language server. IDE
+executable has `--no-engine` flag that switches off the bundled backend. That
+requires you to run the project manager process yourself. Running development
```

Comment:
Adam, you mentioned that the primary way of launching the GUI in your own development workflow is `corepack pnpm dev:gui`. Shouldn't we remove the first sentence:

> You can start IDE with a development version of the language server. IDE
executable has `--no-engine` flag that switches off the bundled backend.

Or move it to the some later section as it can still be useful for testing the whole `.AppImage` & co. during QA...